client/src/helpers/getResourceValue.js

```diff
@@ -0,0 +1,8 @@
+const customResourceValues = {}
+
+export const getResourceValue = (resource, token) => {
+  if (token in customResourceValues) {
+    return customResourceValues[token](resource)
+  }
+  return resource[token] || resource.additional_metadata[token]
+}
```

Comment:
What's this function do? As far as I can tell `customResourceValues` will always be empty, so it'll never return a match. Could you add a comment explaining?

Reply:
Yeah there needs to be some clarification on what the custom fallbacks will be but this is how that will work